module/solutions/helper.js

```diff
@@ -631,11 +654,11 @@ module.exports = class SolutionsHelper {
    * @returns {JSON} - List of solutions based on role and location.
    */
 
-  static forUserRoleAndLocation(bodyData, type, subType = '', programId, pageSize, pageNo, searchText = '') {
+  static forUserRoleAndLocation(bodyData, type, subType = '', programId, pageSize, pageNo, searchText = '',origin,userToken) {
     return new Promise(async (resolve, reject) => {
       try {
         //Getting query based on roles and entity
-        let queryData = await this.queryBasedOnRoleAndLocation(bodyData, type, subType, programId);
+        let queryData = await this.queryBasedOnRoleAndLocation(bodyData, type,origin,userToken, subType, programId);
```

Comment:
@MallanagoudaB token is not required. and if you add new params in between existing param in a function call make sure it is not breaking anything